### PR TITLE
For #6384: Use uri as format argument in error pages description string

### PR DESCRIPTION
**Pull request by mcarare**
---

### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features
error

### After merge
- [ ] **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- [ ] **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### CODE REVIEW

File: components/browser/errorpages/src/main/java/mozilla/components/browser/errorpages/ErrorPages.kt [hunk 1]
```diff
@@ -73,7 +73,7 @@ object ErrorPages {
     ): String {
         val title = context.getString(errorType.titleRes)
         val button = context.getString(errorType.refreshButtonRes)
-        val description = context.getString(errorType.messageRes)
+        val description = context.getString(errorType.messageRes, uri)
```
Comment (Amejia481) [inline components/browser/errorpages/src/main/java/mozilla/components/browser/errorpages/ErrorPages.kt]: Do we know for sure that all description messages must include the `uri`? cc @jonalmeida

File: components/browser/errorpages/src/test/java/mozilla/components/browser/errorpages/ErrorPagesTest.kt [hunk 2]
```diff
@@ -112,10 +112,12 @@ class ErrorPagesTest {
 
         val htmlFilename = "htmlResource.html"
 
+        val uri = "sampleUri"
+
         val errorPage = createUrlEncodedErrorPage(
             context,
             errorType,
-            null,
+                uri,
```
Comment (Amejia481) [inline components/browser/errorpages/src/test/java/mozilla/components/browser/errorpages/ErrorPagesTest.kt]: nit: extra spacing

File: components/browser/errorpages/src/main/java/mozilla/components/browser/errorpages/ErrorPages.kt (hunk 1 above)
Comment (jonalmeida) [reply components/browser/errorpages/src/main/java/mozilla/components/browser/errorpages/ErrorPages.kt]: This should be safe. We do the same thing in the [function above](https://github.com/mozilla-mobile/android-components/pull/6385/files#diff-00750c22f22d2046f9c02e30915d0f31R43).

File: components/browser/errorpages/src/test/java/mozilla/components/browser/errorpages/ErrorPagesTest.kt (hunk 2 above)
Comment (mcarare) [reply components/browser/errorpages/src/test/java/mozilla/components/browser/errorpages/ErrorPagesTest.kt]: My AS was set to use continuation index on function calls on this project. I'm not sure why 🤔

File: components/browser/errorpages/src/main/java/mozilla/components/browser/errorpages/ErrorPages.kt (hunk 1 above)
Comment (mcarare) [reply components/browser/errorpages/src/main/java/mozilla/components/browser/errorpages/ErrorPages.kt]: This is in the doc for java.util.Formatter : 

>  * <p> If there are more arguments than format specifiers, the extra arguments are ignored.

**Comment by jonalmeida** [comment]
bors r=Amejia481

**Comment by jonalmeida** [comment]
bosr retry

**Comment by Amejia481** [comment]
bors retry

**Pull request merged**